modules/metastorage-cache/src/test/java/org/apache/ignite/internal/metastorage/cache/IdempotentCacheVacuumizerTest.java

```diff
@@ -79,6 +82,42 @@ public void tearDown() {
         scheduler.shutdown();
     }
 
+    /**
+     * Tests that ttl and clock skew are properly taken into account while calculating an eviction timestamp.
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testIdempotentCacheVacuumizerTtl() throws Exception {
+        AtomicReference<HybridTimestamp> evictionTimestamp = new AtomicReference<>();
+
+        long ttl = 10L;
+        when(idempotentCacheTtlConfigurationValue.value()).thenReturn(ttl);
+
+        assertEquals(ttl, idempotentCacheTtlConfigurationValue.value());
```

Comment:
What is this check for? Are you testing mockito?

Reply:
Yes, I think I should remove it, this was to make sure that everything works :)